packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -207,20 +219,48 @@ async def download_file_from_s3(
     )
 
     async with ClientSessionContextManager(client_session) as session:
+        store_id = await _resolve_location_id(
+            client_session, user_id, store_name, store_id
+        )
+        file_meta_data: FileMetaDataGet = await _get_file_meta_data(
+            user_id=user_id,
+            s3_object=s3_object,
+            store_id=store_id,
+            client_session=session,
+        )
+
+        if file_meta_data.is_directory and not await r_clone.is_r_clone_available(
+            r_clone_settings
+        ):
+            msg = f"Requested to download directory {s3_object}, but no rclone support was detected"

```

Comment:
Q: do you expect to have r_clone disabled?  If so, you hsould support uploading folders "the old way" (i.e. zip?) instead of failing. 

The idea is to "improve" a working system when r-clone is available, otherwise it works as before (not worse ;-)

Reply:
so this is already the highest level where this error can be launched. this is called by both `ports` and `data_manager(states)` part of nodeports.

Reply:
also not sure if it makes sense to support with RClone disabled at this point